README.md

```diff
@@ -1,8 +1,11 @@
-# wdio-qunit-service
+# @yottaa/wdio-qunit-service
 
-[![npm](https://img.shields.io/npm/v/wdio-qunit-service)](https://www.npmjs.com/package/wdio-qunit-service) [![test](https://github.com/mauriciolauffer/wdio-qunit-service/actions/workflows/test.yml/badge.svg)](https://github.com/mauriciolauffer/wdio-qunit-service/actions/workflows/test.yml)
+[![NPM Version](https://img.shields.io/npm/v/@yottaa/wdio-qunit-service)](https://www.npmjs.com/package/@yottaa/wdio-qunit-service)
+[![test](https://github.com/pmoeller91-yottaa/wdio-qunit-service/actions/workflows/test.yml/badge.svg)](https://github.com/pmoeller91-yottaa/wdio-qunit-service/actions/workflows/test.yml)
```

Comment:
change org to yottaa?

Reply:
Good catch, fixed this and a couple other things I missed. See latest commit. Had to force push to fix some commit messages.